src/gui/static/src/app/components/pages/buy/buy.component.html

```diff
@@ -0,0 +1,26 @@
+<p *ngIf="!otcEnabled">Sorry, otc has currently been disabled!</p>
+<md-card *ngIf="otcEnabled">
+  <md-card-title>Purchase Skycoin</md-card-title>
+  <md-card class="skycoin-details">
+    You can buy Skycoins directly from your wallet using our Skycoin Teller service. The current rate is 0.0002 BTC per
```

Comment:
Will need to show a dynamic rate once we add that api to teller

Reply:
Very true, I have opened an issue for this on the teller project: https://github.com/skycoin/teller/issues/54